### DISCUSSION TITLE
No stats?

### DISCUSSION THREAD

**Discussion by deadrubberboy** [tphakala/birdnet-go#226] [General]
Hello! I see in the config.yaml there is an option to turn on chat logs? Not sure what that is. I would like stats that build over time if possible? Like how many this week, month etc like in birdnet-pi

I am coming form the birdnet-pi Home Assistant add on. It's working but I feel like I prefer it in stand alone docker - so thank you for making this! I was up and running easy peasy. 

note: when I put in my bird weather ID it would crash the container with an error about not being able to read it. I looked at the yaml and saw there was some letters ("int!!" or something) ahead of the ID. I removed those and it popped right up.

**Comment by tphakala** [comment]
There are no stats in Web UI yet, I am not very good at frontend coding so progress on that front has been slow. Also, that BirdWeather ID issue is a bug in settings interface, I'll fix it for next release.

**Comment by matthew73210** [comment]
I was going to have a go a little while ago but life got in the way, there's another discussion on the stats side of things. This weekend if i fine the time i'll try and spin some basic stuff up.
